main.cpp

```diff
@@ -1 +1,32 @@
+#include <iostream> 
+#include <cstdlib>
+#include <iomanip>
+#include "coordinates.h"
+#include <vector> 
 
+using namespace std;
+
+int main() {
+    // loop that reads comma-delimited lines from standard input one line at a time and constructs a vector of coordinates 
```

Comment:
Good work commenting all of this out, makes it a lot easier for me to understand!